src/remote/e-commerce-api/eCommerceClient.ts

```diff
@@ -4,10 +4,10 @@ import axios from 'axios';
 // Very simple, but it also doesn't give us much abstraction
 const eCommerceClient = axios.create({
   withCredentials: true,
-  baseURL: 'http://bizbazaarbizbazaarappeb-backend-env.eba-rzd5q64x.us-east-1.elasticbeanstalk.com/',
+  baseURL: 'http://localhost:8080',
```

Comment:
Please do not commit changes from this file if the only changes are the URLs for local development.